Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1421,8 +1428,15 @@ public WakeOnLanInfo[] GetWakeOnLanInfo()
 
         public async Task<PublicSystemInfo> GetPublicSystemInfo(CancellationToken cancellationToken)
         {
-            var localAddress = await GetLocalApiUrl(cancellationToken).ConfigureAwait(false);
-            var wanAddress = await GetWanApiUrl(cancellationToken).ConfigureAwait(false);
+            var localAddress = await GetLocalApiUrl(cancellationToken).ConfigureAwait(false);            
+            var wanAddress = System.String.Empty;
+            
+            if (string.IsNullOrEmpty(ServerConfigurationManager.Configuration.WanDdns)){
+                wanAddress = await GetWanApiUrl(cancellationToken).ConfigureAwait(false);
```

Comment:
Can we rename this `GetWanApiUrl` function to something like `GetWanApiUrlFromExternal`?

Reply:
Yes, we can rename that one. What about the other methods?

- GetWanApiUrl(string host)
- GetWanApiUrl(IpAddressInfo ipAddress)

Suggestion: they shall be consistent with GetLocalApiUrl(), so no change.